### PR TITLE
AdminX: reroute settings-x to settings

### PR DESCRIPTION
**Pull request by peterzimon**
refs. https://github.com/TryGhost/Product/issues/3949

- before releasing we need to reroute all settings from `settings-x` to `settings`

### CODE REVIEW

File: .github/workflows/browser-tests.yml [hunk 1]
```diff
@@ -77,8 +77,7 @@ jobs:
 
     - name: Build Admin
       if: env.ENVIRONMENT == 'browser-tests-local'
-      working-directory: ghost/admin
-      run: yarn build:dev
+      run: yarn workspace @tryghost/admin-x-settings build && yarn workspace ghost-admin build:dev
```
Comment (daniellockyer) [inline .github/workflows/browser-tests.yml:80]:
```suggestion
      run: yarn nx run ghost-admin:build:dev
```

Comment (daniellockyer) [reply .github/workflows/browser-tests.yml:80]: This is automatically configured to build admin-x-settings if we run it through NX

File: ghost/core/content/themes/source
Comment (daniellockyer) [inline ghost/core/content/themes/source:1]: This needs to not be deleted 😀

Comment (binary-koan) [reply ghost/core/content/themes/source:1]: I specifically told git to reset that change 😠 Thanks for pointing it out!

**Pull request merged**